src/app-layout/utils/use-pointer-events.ts

```diff
@@ -41,9 +41,11 @@ export const usePointerEvents = ({ position, panelRef, handleRef, onResize }: Si
   );
 
   const onDocumentPointerUp = useCallback(() => {
-    if (!panelRef || !panelRef.current) {
+    const panelElement = panelRef?.current;
+    if (!panelElement) {
       return;
     }
+    const document = panelElement.ownerDocument;
```

Comment:
Not really a functional change, but in our demo page setup the `document` points on the wrong frame and we need to account for that

Reply:
I did it this way to make the final diff shorter, but I can also rename if you find it more important